docs/conf/scalar-loki-stack-custom-values.yaml

```diff
@@ -53,3 +53,28 @@ promtail:
                 - __meta_kubernetes_pod_uid
                 - __meta_kubernetes_pod_container_name
               target_label: __path__
+        # -- the `scalar-admin-for-kubernetes` job scrapes all the logs of Scalar Admin for Kubernetes Pods
+        - job_name: scalar-admin-for-kubernetes
+          pipeline_stages:
+            - docker: {}
+            - cri: {}
```

Comment:
This is necessary since many managed Kubernetes cluster is using CRI but not Docker.

Reply:
This is not related to this PR directly, but do we need to add this `cri: {}` configuration in both ScalarDB and ScalarDL configurations?